### PR TITLE
Video player

### PR DESCRIPTION
**Pull request by Abdullahejaz**
Video player with some basic functionality

### CODE REVIEW

File: app/src/main/res/layout/activity_video_demo.xml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            android:id="@+id/fullscreen_content_controls"
+            style="?metaButtonBarStyle"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_gravity="bottom|center_horizontal"
+            android:background="@color/black_overlay"
+            android:orientation="horizontal"
+            tools:ignore="UselessParent">
+
+            <Button
```
Comment (ruraj) [inline app/src/main/res/layout/activity_video_demo.xml:39]: I would remove this and any code related to it in activity. (need to do the same with text reader activity)

File: app/src/main/res/layout/activity_video_demo.xml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="#0099cc"
+    tools:context="io.github.antishake.VideoPlayer">
+
+    <!-- The primary full-screen view. This can be replaced with whatever view
+         is needed to present your content, e.g. VideoView, SurfaceView,
+         TextureView, etc. -->
+    <TextView
```
Comment (ruraj) [inline app/src/main/res/layout/activity_video_demo.xml:11]: I would also remove this. (need to do the same with text reader activity)

File: app/src/main/res/layout/activity_video_demo.xml (hunk 1 above)
Comment (saranya-shanmugam) [reply app/src/main/res/layout/activity_video_demo.xml:39]: We need to modify this xml to show the video in full screen. So, I thought I will edit it altogether.

Comment (ruraj) [reply app/src/main/res/layout/activity_video_demo.xml:39]: Okay

**Review by ruraj — APPROVED** [review]

**Pull request merged**